bbb-lti/build.gradle

```diff
@@ -126,3 +145,7 @@ assets {
   minifyJs = true
   minifyCss = true
 }
+
+springBoot {
+  mainClassName = 'org.bigbluebuttoni.Application'
```

Comment:
```suggestion
  mainClassName = 'org.bigbluebutton.Application'
```
Typo?

Reply:
I think this is how it should be considering the class package statement looks like this
`package org.bigbluebuttoni`

https://github.com/bigbluebutton/bigbluebutton/blob/8404048cfd35280d2dd427fc0c2e93790760c3f7/bbb-lti/grails-app/init/org/bigbluebutton/Application.groovy#L19